app/src/main/java/com/github/libretube/helpers/ProxyHelper.kt

```diff
@@ -10,6 +10,8 @@ import okhttp3.OkHttpClient
 import okhttp3.Request
 
 object ProxyHelper {
+    private val client by lazy { OkHttpClient.Builder().build() }
```

Comment:
Why not use the cronet client here? :thinking: 
We have an adaptor for Cronet's engine for OkHttp